sc/src/test/java/uk/ac/ebi/atlas/download/FileDownloadControllerWIT.java

```diff
@@ -134,18 +134,21 @@ void downloadArchiveForValidExperimentAccessions() throws Exception {
                 MessageFormat.format(
                         ARCHIVE_NAME, EXPERIMENT_ACCESSION_LIST.size(), "experiment");
         this.mockMvc.perform(get(ARCHIVE_DOWNLOAD_LIST_URL)
-                .param("accession", EXPERIMENT_ACCESSION_LIST.toString().replace("[", "").replace("]", "")))
+                .param("accession", EXPERIMENT_ACCESSION_LIST.get(0))
+                .param("accession", EXPERIMENT_ACCESSION_LIST.get(1)))
                 .andExpect(status().isOk())
-                .andExpect(
-                        header().string(HttpHeaders.CONTENT_DISPOSITION, "attachment;filename=" + expectedArchiveName))
-                .andExpect(content().contentType("application/zip"));
+                .andExpect(content().contentType("application/zip"))
+                .andExpect(header().string("Content-Length", String.valueOf(EXPERIMENT_ACCESSION_LIST.size() * 10)))
```

Comment:
Why does the Content-Length header need to match that specific value? How does that indicate the validity of the zip file?